docs/deployment/deploy-without-docker/Deploying.md

```diff
@@ -1,131 +1,29 @@
 # Deploying the Web Application
 
-## Prepare the global configuration file
+## Before running cbioportal backend
 
-The portal is configured using a global configuration file, `application.properties`. An example file is available in the `src/main/resources` folder. Use it as a template to create your own:
-
-```
-cd src/main/resources
-cp application.properties.EXAMPLE $HOME/cbioportal/application.properties
-```
```

Comment:
i think is still relevant right?

https://github.com/cBioPortal/cbioportal/blob/master/src/main/resources/application.properties.EXAMPLE

Or slightly changed maybe to mention that one can use the `EXAMPLE` file as a starting point?

Reply:
ah i see you moved this -nvm